content/id/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -0,0 +1,245 @@
+    ```
+
+3. Jalankan ulang kubelet. Pada Fedora, kamu dapat menjalankan:
+
+    ```shell
+    # Jalankan perintah ini pada Node tempat kubelet berjalan
+    systemctl restart kubelet
+    ```
+
+## Mengobservasi perilaku Pod statis
+
+Ketika kubelet berjalan, secara otomatis akan menjalankan semua Pod statis yang terdefinisi.
+Ketika kamu mendefinisikan Pod statis dan menjalankan ulang kubelet, Pod statis yang baru
+akan dijalankan.
+
+Kamu dapat melihat kontainer yang berjalan (termasuk Pod statis) dengan menjalankan (pada Node):
+```shell
+# Jalankan perintah ini pada Node tempat kubelet berjalan
+docker ps
+```
+
+Hasilnya akan seperti berikut:
+
+```
+CONTAINER ID IMAGE         COMMAND  CREATED        STATUS         PORTS     NAMES
+f6d05272b57e nginx:latest  "nginx"  8 minutes ago  Up 8 minutes             k8s_web.6f802af4_static-web-fk-node1_default_67e24ed9466ba55986d120c867395f3c_378e5f3c
+```
+
+Kamu dapat melihat Pod mirror pada API server:
+
+```shell
+kubectl get pods
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          2m
+```
+
+{{< note >}}
+Pastikan kubelet memiliki izin untuk membuat Pod mirror pada API server. Jika tidak,
+pembuatannya akan ditolak oleh API server. Lihat
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/).
+{{< /note >}}
+
+
+{{< glossary_tooltip term_id="label" text="Label" >}} dari Pod statis
+akan dibuat juga pada Pod mirror. Kamu dapat menggunakan label tersebut
+seperti biasa menggunakan {{< glossary_tooltip term_id="selector" text="selectors" >}},
+atau yang lainnya.
+
+Kamu dapat mencoba untuk menggunakan `kubelet` untuk menghapus Pod mirror pada API server,
+namun kubelet tidak akan menghapus Pod statis:
+
+```shell
+kubectl delete pod static-web-my-node1
+```
+```
+pod "static-web-my-node1" deleted
+```
+Kamu akan melihat Pod tetap bejalan:
+```shell
+kubectl get pods
+```
+```
+NAME                       READY     STATUS    RESTARTS   AGE
+static-web-my-node1        1/1       Running   0          12s
+```
+
+Kembali ke Node tempat kubelet berjalan, kamu dapat mencoba menghentikan kontainer
+Docker secara manual.
+Kamu akan menlihat, setelah beberapa waktu, kubelet akan tahu dan akan menjalankan ulang Pod
+secara otomatis:
+
+```shell
+# Jalankan perintah ini pada Node tempat kubelet berjalan
+docker stop f6d05272b57e # ganti dengan ID pada kontainermu
+sleep 20
+docker ps
+```
+```
+CONTAINER ID        IMAGE         COMMAND                CREATED       ...
+5b920cbaf8b1        nginx:latest  "nginx -g 'daemon of   2 seconds ago ...
+```
+
+## Penambahan dan pengurangan secara dinamis pada Pod statis
+
+Kubelet yang berjalan akan secara berkala memindai direktori konfigurasi
+(`/etc/kubelet.d` pada contoh kita) untuk melakukan perubahan dan penambahan/pengurangan
+Pod sesuai dengan penambahan/pengurangan berkas pada direktori ini.
```

Comment:
try to avoid writing kubelet with uppercase

```suggestion
Direktori konfigurasi (`/etc/kubelet.d` pada contoh kita) akan dipindai secara berkala oleh kubelet
untuk melakukan perubahan dan penambahan/pengurangan
Pod sesuai dengan penambahan/pengurangan berkas pada direktori tersebut.
```